.kitchen.cloud.yml

```diff
@@ -19,13 +19,11 @@ platforms:
   driver_config:
     username: centos
     image_ref: "CentOS 6.6"
-- name: centos-7.1
-  driver_plugin: openstack
-  driver_config:
-    username: centos
-    image_ref: "CentOS 7.1"
 
 suites:
   - name: default
     run_list:
       - recipe[formsender::default]
+      - recipe[formsender::python]
+      - recipe[formsender::build_essential]
```

Comment:
`::build_essential` and `::python` should come before `::default`, or the attributes they change will already be used by recipes included via `::default` before they get changed.